workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/components/DashboardTable/PurchasedAppsDashboardTableRow.tsx

```diff
@@ -136,7 +139,7 @@ export function PurchasedAppsDashboardTableRow({
 				</div>
 			</ClayTable.Cell>
 
-			<ClayTable.Cell>
+			<ClayTable.Cell onClick={(event) => event.stopPropagation()}>
 				<DropDown
 					trigger={
 						<ClayButton displayType="secondary">
```

Comment:
If possible, could you to add the lib i18n for this file?

 - Manage
 - Create License Key
 - Access Console
 - Download App
 - This order must be completed before downloading this app.
 - The order must be completed before licensing this app.